modules/vulnerabilities/unix/http/lucee_rce/secgen_metadata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<vulnerability xmlns="http://www.github/cliffe/SecGen/vulnerability"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.github/cliffe/SecGen/vulnerability">
+
+  <name>Lucee Administrator imgProcess.cfm Arbitrary File Write</name>
+  <author>James Davis</author>
+  <module_license>MIT</module_license>
+  <description>
+    This module exploits an arbitrary file write in Lucee Administrator's
+    imgProcess.cfm file to execute commands as the Tomcat user.
+  </description>
+
+  <type>http</type>
+  <type>in_the_wild</type>
+  <privilege>user_rwx</privilege>
+  <access>remote</access>
+  <platform>linux</platform>
+  <difficulty>low</difficulty>
+
+  <read_fact>port</read_fact>
+  <read_fact>strings_to_leak</read_fact>
+  <read_fact>leaked_filenames</read_fact>
+
+  <default_input into="port">
+    <generator module_path=".*/random_unregistered_port" />
+  </default_input>
+
+  <!-- flags or other secrets exposed after exploitation -->
+  <default_input into="strings_to_leak">
+    <generator type="flag_generator" />
+  </default_input>
+
+  <default_input into="leaked_filenames">
+    <value>flag</value>
+  </default_input>
+
+  <default_input into="leaked_username">
```

Comment:
I suspect you are using the wrong parameter name. Sorry it could be clearer, but leaked username would be one leaked by the vuln to the attacker